doc/development_guide/contributor_guide/governance.rst

```diff
@@ -0,0 +1,48 @@
+==============
+ Governance
+==============
+
+How to become a triager ?
+-------------------------
+
+Contribute for more than 3 releases consistently. Do not be too opinionated, follow the code of conduct without
+requiring emotional works from the maintainers. It does not mean that disagreements are impossible, only that arguments
+should stay technical and reasonable so the conversation is civil and productive. Get unanimous approval or neutral
+agreement from current maintainers.
+
+How to become a maintainer ?
+----------------------------
+
+Contribute two big code merge requests over multiple releases (for example like one checker in 2.13 and one
+complicated bug fixes in 2.14). Otherwise contributing for more than 3 releases consistently with
+great technical and interpersonal skills. Become an expert in a part of the code that is not maintained
+well at the moment (if we feel we can't merge something without your review, you're going to be able to
+merge those yourself soon). Get unanimous approval or neutral agreement from current maintainers.
+
+
+How to get admin rights ?
+-------------------------
+
+Contribute for several hundreds of commits over a long period of time with excellent interpersonal skills
+and code quality. Get unanimous approval or neutral agreement from current admins.
```

Comment:
If there are enough admins (and not just one) like it's the case at the moment, I'm not sure it's necessary to add any new ones. Almost all common tasks can be accomplished as a maintainer as well.

Furthermore, this should only be on suggestion by an existing admin.